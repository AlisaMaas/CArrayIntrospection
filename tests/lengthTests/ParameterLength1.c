/**
* This tests traditional parameter-length.
**/
int foo(int* array, int x){
	int sum = 0;
	for(int i = 0;  x > i;  i++){
	    sum += array[4];
		sum += array[i];
	}
	return sum;
}
