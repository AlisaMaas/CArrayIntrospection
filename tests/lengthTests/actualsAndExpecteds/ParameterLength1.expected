Analyzing foo
	Argument array has max index argument x
	Argument x has unknown max index.
